README.md

```diff
@@ -47,6 +47,8 @@ source tree), the backup is considered *complete*.
 
     conserve backup /backup/home.cons ~
 
+Provide as option `--exclude` a list of [glob patterns](https://en.wikipedia.org/wiki/Glob_(programming) to exclude files or folders from backup.
```

Comment:
Could you link to something giving the specific glob syntax supported by `GlobSet`?

Reply:
We should also mention this in README.md.

Reply:
Can you rephrase it to 

    Use `--exclude GLOB` to exclude files or directories from the backup.